src/psyclone/domain/lfric/__init__.py

```diff
@@ -72,6 +73,7 @@
     'KernelInterface',
     'KernStubArgList',
     'LFRicArgDescriptor',
+    'LFRicCollection'
```

Comment:
`pylint` says: `__init__.py:76:4: E0603: Undefined variable name 'LFRicCollectionLFRicConstants' in __all__ (undefined-all-variable)`. 

I think there should be a comma :smile: